src/main/java/com/epam/machine/repository/ClientRepositoryImpl.java

```diff
@@ -0,0 +1,90 @@
+package com.epam.machine.repository;
+
+import com.epam.machine.entity.Client;
+import lombok.Builder;
+import lombok.Data;
+
+import java.sql.*;
```

Comment:
Do not use wildcards in import section, adjust setting in IDE to not use `*`. Settings->Editor->Code Style->Java-> Imports